src/MCPClient/lib/clientScripts/removeUnneededFiles.py

```diff
@@ -2,21 +2,32 @@
 
 import os
 import sys
+import shutil
 
 # databaseFunctions requires Django to be set up
 import django
 django.setup()
 # archivematicaCommon
 from custom_handlers import get_script_logger
 from databaseFunctions import fileWasRemoved
+import ConfigParser
 
-REMOVEABLE_FILES = ["Thumbs.db", "Icon", u"Icon\u000D"]
+clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
+config = ConfigParser.SafeConfigParser()
+config.read(clientConfigFilePath)
+try:
+    removableFiles = [e.strip() for e in config.get('MCPClient', 'removableFiles').split(',')]
+except:
```

Comment:
What exceptions are you expecting here?  `ConfigParser.NoOptionError` ? Please catch only the exceptions you're expecting.